### PR TITLE
Fix VRT failures in CI

### PR DESCRIPTION
**Pull request by tofumatt**
## Summary

Addresses issue:

- #9528

## PR Author Checklist

- [ ] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 5.2 and PHP 7.4.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [ ] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [ ] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [ ] Run the code.
- [ ] Ensure the acceptance criteria are satisfied.
- [ ] Reassess the implementation with the IB.
- [ ] Ensure no unrelated changes are included.
- [ ] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [ ] Ensure there is a QA Brief.
- [ ] Ensure there are no unexpected significant changes to file sizes.

## Merge Reviewer Checklist

- [ ] Ensure the PR has the correct target branch.
- [ ] Double-check that the PR is okay to be merged.
- [ ] Ensure the corresponding issue has a ZenHub release assigned.
- [ ] Add a changelog message to the issue.

### CODE REVIEW

**Comment by eugene-manuilov** [comment]
Also, there are two `ca-certificates` packages are being installed. We need to cleanup installed packages to have no duplicates.

**Comment by eugene-manuilov** [comment]
@benbowler are all those `lib*` packages that you added in the dockerfile file really needed? Or it was copy/pasted from somewhere?

**Comment by benbowler** [comment]
> @benbowler are all those `lib*` packages that you added in the dockerfile file really needed? Or it was copy/pasted from somewhere?

@eugene-manuilov they are from the [puppeteer documentation linked in the code comment](https://github.com/puppeteer/puppeteer/blob/main/docs/troubleshooting.md#running-puppeteer-on-gitlabci):

> In gitlab-ci.yml we need to install some packages to make it possible to launch headless Chrome in your docker env.

File: docker/backstopjs/Dockerfile [hunk 1]
```diff
@@ -1,28 +1,31 @@
-FROM alpine:3.17
+FROM node:16.14.2-bullseye
 
-ARG BACKSTOPJS_VERSION=6.1.1
+# Prevent intractive prompts from apt-get.
+ARG DEBIAN_FRONTEND=noninteractive
 
+ARG BACKSTOPJS_VERSION=6.1.1
```
[the remark this answers is not in the document]
Comment (tofumatt) [reply docker/backstopjs/Dockerfile:6]: Oh yes, sorry, I missed that 👍🏻

**Comment by tofumatt** [comment]
Superseded by #9721.

**Pull request closed without merging**